src/main/groovy/com/epam/petclinic/plugin/ArtifactoryPlugin.groovy

```diff
@@ -0,0 +1,50 @@
+package com.epam.petclinic.plugin
+
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+import org.gradle.api.publish.maven.MavenPublication
+import org.gradle.api.tasks.bundling.Jar
+
+/**
+ * The ArtifactoryPlugin deploy artifacts and build information to Artifactory.
+ *
+ * Date: 5/4/2017
+ *
+ * @author Stanislau Halauniou
+ */
+class ArtifactoryPlugin implements Plugin<Project> {
```

Comment:
I suggest do it in another pull, like adding pmd, findbug, idea formatting and other plugins. Let's discuss priorities later.